engines/wintermute/ad/ad_object_3d.cpp

```diff
@@ -629,18 +629,14 @@ bool AdObject3D::getBonePosition3D(const char *boneName, Math::Vector3d *pos, Ma
 		return false;
 	}
 
-	if (!offset) {
-		*offset = Math::Vector3d(0, 0, 0);
-	}
-
 	Math::Matrix4 bonePosMat = *boneMat * _worldMatrix;
+	*pos = Math::Vector3d(0.0f, 0.0f, 0.0f);
```

Comment:
This assumes that pos is not a nullptr, if pos can't be a nullptr, then it should be a reference, not a pointer, to make it impossible to pass nullptr.